test/fixtures/for/in-auto-const/actual.js

```diff
@@ -1 +1 @@
-for k in {}: k
+for const k in {}: k
```

Comment:
My thinking was to simply eliminate the `auto-const` folder (once I fix `for..of`) because we don't auto-const anymore, yes?